packages/@core.keystone/tasks.js

```diff
@@ -155,11 +173,28 @@ async function createWorker (keystoneModule) {
 
     await taskQueue.isReady()
     taskLogger.info({ message: 'Worker: ready to work!' })
+    const cronTasksNames = [...CRON_TASKS.keys()]
+    if (cronTasksNames.length > 0) {
+        taskLogger.info({ message: 'Worker: add repeatable tasks!', names: cronTasksNames })
+        cronTasksNames.forEach((name) => {
+            const fn = TASKS.get(name)
+            fn.delay()
+        })
+    }
```

Comment:
It's unclear, how a task is set as repeatable. There is no mention of this.
Could you please write an instruction

Reply:
I think we can discuss it on out tech tolk